### DISCUSSION TITLE
21-06-22 일일회고

### DISCUSSION THREAD

**Discussion by GleamingStar** [d-h-k/issue-tracker#91] [회고]
🔢

**Comment by GleamingStar** [comment]
오늘은 뷰를 마무리하고, 스타브가 만들어주신 #17 issue API를 적용 시도했습니다
body도 comment에 넣어주시고, 일괄 닫기,열기 기능도 있는 등 꼼꼼하게 만들어주셔서 금방 될 것 같습니다!

**Comment by eeeesong** [comment]
오늘은 이슈 작성 화면을 구현했습니다!
마크다운 라이브러리를 적용하고, 기존 뷰를 재사용해서 코드를 효율적으로 작성하는 부분에 집중했습니다 
이제 수료식 빼면 이틀남았네요 세상에나!

**Comment by reminderclock** [comment]
오늘은 마일스톤 추가, 삭제, 디자인 수정을 했습니다. 밀린 성 코드리뷰도 조금 진행하였고요; 내일은 마일스톤 편집, 마일스톤 달성률 작업하면 될것같습니다.
